README.md

```diff
@@ -1,5 +1,45 @@
-[![Build Status](https://travis-ci.org/punkave/apostrophe.svg?branch=master)](https://travis-ci.org/punkave/apostrophe)
+# [<img src="./ApostropheCMS_logo.png" height="140" title="ApostropheCMS" />](http://apostrophecms.org/)
+
+ApostropheCMS is a content management system (CMS) for Node.js. It supports in-context editing, schema-driven content types, flexible widgets and a great deal more. This module contains everything necessary to build a website with ApostropheCMS.
+
+Please see the [ApostropheCMS documentation](http://apostrophecms.org/docs) for detailed information about building sites with ApostropheCMS.
+
+## Getting Started
+
+The [ApostropheCMS documentation](http://apostrophecms.org/docs/tutorials/getting-started/index.html) will walk you through some simple steps to get your [local environment setup](http://apostrophecms.org/docs/tutorials/getting-started/setting-up-your-environment.html) with the technology stack that runs ApostropheCMS (Node.js, NPM, MongoDB, ImageMagick) along with installing ApostropheCMS itself.
```

Comment:
"set up", not "setup"

Reply:
I think you can leave "NPM" off the list because installing Node is enough and they might try to find it